utilities/utils.py

```
@@ -262,6 +286,35 @@ def gen_network_map_list(
+    Returns:
+        list[dict[str, str]]: Reordered networks with custom_pod_network first
+
+    Raises:
+        ValueError: If custom_pod_network not found in networks
+    """
+    pod_net = next((net for net in networks if net["name"] == custom_pod_network), None)
+    if not pod_net:
+        available = [net["name"] for net in networks]
+        raise ValueError(
+            f"Network '{custom_pod_network}' specified as custom_pod_network not found. Available networks: {available}"
+        )
```

Ruff (0.15.13)

[warning] Avoid specifying long messages outside the exception class

(TRY003)